response_operations_ui/views/case.py

```diff
@@ -40,22 +39,23 @@ def get_response_statuses(ru_ref, error=None):
     logger.info("Retrieving response statuses", ru_ref=ru_ref)
     short_name = request.args.get("survey")
     period = request.args.get("period")
+
     completed_respondent = ""
 
     survey = survey_controllers.get_survey_by_shortname(short_name)
+
     exercises = collection_exercise_controllers.get_collection_exercises_by_survey(survey["id"])
     exercise = collection_exercise_controllers.get_collection_exercise_from_list(exercises, period)
+
     reporting_unit = party_controller.get_business_by_ru_ref(ru_ref)
 
-    # The possible_transitions_for_case variable is a dict that contains the action as the key (EQ_LAUNCH for example)
-    # and the value is the new state it will be in (e.g., COMPLETE, INPROGRESS, etc).
-    # The allowed_transitions_for_case are the statuses that we allow a case to switch to.  We currently only allow
-    # a subset of them via the UI as case can't handle switching between certain states correctly.
-    possible_transitions_for_case = case_controller.get_available_case_group_statuses_direct(exercise["id"], ru_ref)
-    allowed_transitions_for_case = {
+    statuses = case_controller.get_available_case_group_statuses_direct(exercise["id"], ru_ref)
```

Comment:
Why'd you change the names back?  I made them more wordy to actually explain what they are.  `statuses` and `available_statuses` don't mean as much as `possible_transitions_for_case` and `allowed_transitions_for_case` I think

Reply:
I can change it back